monkey/infection_monkey/post_breach/actions/hide_files.py

```diff
@@ -30,9 +30,13 @@ def run(self):
                 windows_cmd=windows_cmds,
             )
             super(HiddenFiles, self).run()
+
         if is_windows_os():  # use winAPI
             result, status = get_winAPI_to_hide_files()
-            PostBreachTelem(self, (result, status)).send()
+            # no command here, used WinAPI
+            self.pba_data.append(PostBreachData(self.name, "", (result, status)))
```

Comment:
For this and similar cases wouldn't be better if we send something like `service_used`? I mean as a user I will what to know what is used when I check telemetries.

Reply:
Yes, it is. Check the discussion on Slack regarding this. We'll get to this during the telemetry refactor. I'm not sure it's worth investing time in this right now since we don't know what the new telemetry interface is going to look like.